### PR TITLE
Unify busybox variants

### PR DESCRIPTION
**Pull request by nirs**
We had 4 copies of the busybox deployment and pvcs. This change unifies the configuration:

- Use same command in the deployment
- Use same pvc size (5Gi)
- Use same list syntax for pvc accessModes

And remove duplicated files using kustomization.yaml.

### CODE REVIEW

File: busybox/busybox-pvc.yaml [hunk 1]
```diff
@@ -6,8 +6,9 @@ metadata:
...
 spec:
-  accessModes: [ "ReadWriteOnce" ]
-  storageClassName: "rook-ceph-block"
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: rook-ceph-block
   resources:
     requests:
-      storage: 1Gi
+      storage: 5Gi
```
Comment (ShyamsundarR) [inline busybox/busybox-pvc.yaml:14]: Why the change in size?

**Review by ShyamsundarR — COMMENTED** [review]
This requires some attention from documentation that is already present for the downstream product (which uses a forked version of this repository). @netzzer if the layout changes as per this PR, what is the documentation impact?

File: busybox/busybox-pvc.yaml (hunk 1 above)
Comment (nirs) [reply busybox/busybox-pvc.yaml:14]: All the other variants use 5Gi - we can change this in the kustomization but I don't see a reason to invest time in keeping different size. If there is no good reason to use 5Gi we can change all of them to use 1Gi.

**Comment by nirs** [comment]
> This requires some attention from documentation that is already present for the downstream product (which uses a forked version of this repository). @netzzer if the layout changes as per this PR, what is the documentation impact?

The docs mention the name of the app (e.g. busybox-odr) - which is not changed, so it should
work in the same way. I did not test it with openshift.

File: busybox/busybox-pvc.yaml (hunk 1 above)
Comment (nirs) [reply busybox/busybox-pvc.yaml]: I changed all pvcs to 1Gi since it makes more sense. If there is real need to use different size we can modify it later, maybe only on the specific variant that need this.

**Comment by ShyamsundarR** [comment]
> > This requires some attention from documentation that is already present for the downstream product (which uses a forked version of this repository). @netzzer if the layout changes as per this PR, what is the documentation impact?
> 
> The docs mention the name of the app (e.g. busybox-odr) - which is not changed, so it should work in the same way. I did not test it with openshift.

If you can confirm this behavior (even non Openshift is fine, as long as the instructions as per doc are followed) and if it is broken, then a downstream tracker for the same to fix docs by the next release would help. As that way we can take in this change and fix documentation subsequently as the downstream variant is released.

**Comment by nirs** [comment]
Checking the docs[1] shows that we always use `busybox-odr*` so these changes should not affect users using these samples.

We need some minor updates like up example usage, showing now 5Gi pvc size, but with this change the size is 1Gi.

```
NAME                                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                AGE
persistentvolumeclaim/busybox-pvc   Bound    pvc-c732e5fe-daaf-4c4d-99dd-462e04c18412   5Gi    
```

Where to open a documentation issue/bug? The docs suggests 2 options[2] - feedback button or opening a bug in Bugzilla. There is no "Feedback button in the upper right corner of the document." so I guess the only way is to open a bug. Maybe the feedback button appears only when you are logged in, but I don't see why log in should be required to read the docs or report bugs.

Also do we have upstream docs repo somewhere?

[1] https://access.redhat.com/
documentation/en-us/red_hat_openshift_data_foundation/4.13/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index
[2] https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html-single/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/index#providing-feedback-on-Red-Hat-documentation_common
